network-api/networkapi/buyersguide/templates/about/about_nav.html

```diff
@@ -1,12 +1,13 @@
-{% load bg_nav_tags i18n %}
+{% load bg_nav_tags i18n wagtailroutablepage_tags %}
 
 {% with request.get_full_path as current_url %}
-{% url 'about-why-view' as about_why_url %}
-{% url 'how-to-use-view' as how_to_use_url %}
-{% url 'min-security-view' as minimum_security_url %}
-{% url 'methodology-view' as methodology_url %}
-{% url 'contact-view' as contact_url %}
-{% url 'press-view' as press_url %}
+{% routablepageurl page "methodology-view" as methodology_url2 %}
+{% routablepageurl page 'about-why-view' as about_why_url %}
+{% routablepageurl page 'how-to-use-view' as how_to_use_url %}
+{% routablepageurl page 'min-security-view' as minimum_security_url %}
+{% routablepageurl page 'methodology-view' as methodology_url %}
+{% routablepageurl page 'contact-view' as contact_url %}
+{% routablepageurl page 'press-view' as press_url %}
```

Comment:
why is methodology-view in the list twice?

Reply:
👀 Because I forgot to remove it when I was testing a thing haha. Thanks for catching this! Removed.